test/spec.index.js

```diff
@@ -388,6 +388,40 @@ describe('Template Mixins', function () {
                 }));
             });
 
+            describe('form-group-year class', function () {
+                var dayCall;
+                var monthCall;
+                var yearCall;
+
+                beforeEach(function () {
+                    middleware(req, res, next);
+                    res.locals['input-date']().call(res.locals, 'field-name');
+
+                    render.called;
+
+                    dayCall = render.getCall(2);
+                    monthCall = render.getCall(4);
+                    yearCall = render.getCall(6);
+                });
+
+                it('not set with day', function () {
+                    dayCall.should.not.have.been.calledWith(sinon.match({
+                        formGroupClassName: 'form-group-year'
+                    }));
+                });
+                it('not set with month', function () {
+                    monthCall.should.not.have.been.calledWith(sinon.match({
+                        formGroupClassName: 'form-group-year'
+                    }));
+                });
+                it('not set with month', function () {
+                    yearCall.should.have.been.calledWith(sinon.match({
+                        formGroupClassName: 'form-group-year'
+                    }));
+                });
+
+            });
```

Comment:
I'm not sure why this is split out into a describe block with multiple tests? it seems like a bit of unnecessary overhead to setup and teardown 3 times when a single `it('form-group-year class is only set on the year input', () => {...});` would suffice.

Pretty much exactly as per the above test.